Source/WebCore/dom/TrustedType.h

```diff
@@ -53,6 +53,10 @@ WEBCORE_EXPORT ExceptionOr<String> trustedTypeCompliantString(TrustedType, Scrip
 
 WEBCORE_EXPORT ExceptionOr<String> requireTrustedTypesForPreNavigationCheckPasses(ScriptExecutionContext&, const String& urlString);
 
+ExceptionOr<String> trustedScriptCompliantStringFromVariant(ScriptExecutionContext&, std::variant<RefPtr<TrustedScript>, String>&&, const String& sink);
+
+ExceptionOr<String> trustedScriptURLCompliantStringFromVariant(ScriptExecutionContext&, std::variant<RefPtr<TrustedScriptURL>, String>&&, const String& sink);
```

Comment:
Not sure this needs to be in a header. Can these helper functions live in `HTMLScriptElement.cpp` instead?

Reply:
These will end up being used in other places so I think they're better off in this file.